pyproject.toml

```diff
@@ -85,6 +88,9 @@ pytest = "==8.0.2"
 pytest-cov = "==4.1.0"
 pytest-mock = "==3.12.0"
 
+[tool.poetry.scripts]
+install_nltk_data = 'ansys.fluent.core.install_nltk_data:install'
+
```

Comment:
@mkundu1 Here we have added a script `install_nltk_data.py` which runs `install` function to install NLTK data.